File: abc/abc144/abc144_b.cs
// https://atcoder.jp/contests/abc144/submissions/8152532

using System;

namespace CSharp
{
    class Program
    {
        public static void Main()
        {
            var N = int.Parse(Console.ReadLine());

            var result = false;
            for (int i = 1; i <= 9; i++)
            {
                for (int j = 1; j <= 9; j++)
                {
                    if (i * j == N)
                    {
                        result = true;
                        break;
                    }
                }
            }
            Console.WriteLine(result ? "Yes" : "No");
        }
    }
}

File: abc/abc144/abc144_b.fs
// https://atcoder.jp/contests/abc144/submissions/12054821

let N = stdin.ReadLine() |> int

[ 1..9 ]
|> List.filter (fun x -> N % x = 0)
|> List.filter (fun x -> N / x < 10)
|> List.length
|> fun x ->
    if x > 0 then "Yes"
    else "No"
|> stdout.WriteLine
